worker/uniter/uniter.go

```diff
@@ -346,6 +346,7 @@ func (u *Uniter) loop(unitTag names.UnitTag) (err error) {
 		localState := resolver.LocalState{
 			CharmURL:             charmURL,
 			CharmModifiedVersion: charmModifiedVersion,
+			UpgradeSeriesStatus:  params.UnitNotStarted,
```

Comment:
This is the only place where I've seen UnitNotStarted, is this exposed somehow?

Reply:
Yes, the Uniter API starts in Unit not started state. Yet, the act of creating a lock indicates that the user wants to start the process of upgrading the units. Therefore the controller corresponding remote state is create in the `core.UnitStarted` state. This will cause the hook to fire once the user issues a `juju upgrade-series prepare`